### PR TITLE
Worked on typography

### PR DESCRIPTION
**Pull request by mperrotti**
See the type spec in `docs/build/pages/type-spec.html`

### CODE REVIEW

File: docs/pages/type-spec.html [hunk 1]
```diff
@@ -0,0 +1,586 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+
+	<title>Sassquatch 2: Basic view example</title>
+
+	<!-- Whitney -->
+	<link rel="stylesheet" href="http://static1.meetupstatic.com/fonts/whitney.css" type="text/css" />
```
Comment (akdetrick) [inline docs/pages/type-spec.html]: replace with
```
<link rel="stylesheet" href="css/font/font.css">
```

Comment (akdetrick) [inline docs/pages/type-spec.html]: delete :put_litter_in_its_place:

File: sass/_globals.scss [hunk 2]
```diff
@@ -103,8 +103,9 @@ $W_medium : 600;
 $W_bold   : 800;
 
 // Leading
-$line-height       : 1.25;
-$line-height-small : 1.4;
+$line-height-large : 1.25;
+$line-height       : 1.45;
```
Comment (akdetrick) [inline sass/_globals.scss]: :raised_hands:

File: sass/core/type/_body-text.scss [hunk 3]
```diff
@@ -336,6 +322,9 @@ numbers/bullets to a list.
 %_listStyleBase {
 	list-style-position: outside;
 	padding-left: 1.33em; /* indent by default */
+	@include atMediaUp(huge){
+		padding-left: inherit; /* indent by default */
```
Comment (akdetrick) [inline sass/core/type/_body-text.scss]: not sure i understand the nature of this rule

Comment (mperrotti) [inline sass/core/type/_body-text.scss]: This was to have hanging indents on wider screens. If you disagree for some reason, we can talk about getting rid of it.
screen shot 2016-03-01 at 1 34 51 pm
screen shot 2016-03-01 at 1 35 06 pm

Comment (akdetrick) [inline sass/core/type/_body-text.scss]: :100:

**Pull request merged**